uludag-universitesi-sosyal-bilimler-enstitusu-full-note-with-ibid.csl

```diff
@@ -14,16 +14,22 @@
     <category citation-format="note"/>
     <category field="social_science"/>
     <summary xml:lang="tr">Uludağ Üniversitesi Sosyal Bilimler Enstitüsü'nde yüksek lisans ve doktora tezi yazan öğrencilerimiz için tez yazım kılavuzuna uygun olarak hazırlanmıştır.</summary>
-    <updated>2018-04-29T22:39:28+00:00</updated>
+    <updated>2018-06-29T15:32:43+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale xml:lang="tr">
     <terms>
       <term name="editor" form="verb-short">ed.</term>
       <term name="translator" form="verb-short">çev.</term>
       <term name="translator" form="short">çev.</term>
-      <term name="editortranslator" form="verb-short">ed. &amp; çev.</term>
-      <term name="editortranslator" form="verb">ed. &amp; çev.</term>
+      <term name="editortranslator" form="verb-short">
```

Comment:
I don't mind this, but FWIW, it's not doing anything. If single/multiple are identical a single term will do.

Reply:
@adam3smith , 
You are absolutely right. Shall I update it now?

Reply:
no need since it doesn't change things. Just revert this next time you update the styles.